lib/python/rose/apps/rose_ana.py

```diff
@@ -85,32 +85,53 @@ def create(self):
             status TEXT,
             comparison TEXT,
             PRIMARY KEY(app_task))
-            """)
-        # And this one stores the completion status of the task
-        conn.execute(
             """
+        self.execute_sql_retry(conn, (sql_statement,))
+        # And this one stores the completion status of the task
+        sql_statement = """
```

Comment:
Hmm... I can change the name of the second one if you're insisting but I don't see that it's a huge problem - the statement execution can't fail and the variable is used in both cases immediately on the next line.  (It's similar to the fairly common use of a variable called something like "msg" for error-strings to be passed to an exception)